[TRAFFIC-11003] Fix parsing of AZ out of bootstrap and broker URLs for Azure PrivateL…

rajukumar-cflt opened the pull request:
…ink clusters

- Fix case of a char in json query expression
- For non-glb urls that separate lkc, broker octets, and AZ with dots, normalize them first by replacing bottom two dots with dashes before extracting AZ out of them

rajukumar-cflt committed:
[TRAFFIC-11003] Fix parsing of AZ out of bootstrap and broker URLs for Azure PrivateLink clusters

- Fix case of a char in json query expression
- For non-glb urls that separate lkc, broker octets, and AZ with dots, normalize them first by replacing bottom two dots with dashes before extracting AZ out of them

michaelbor reviewed (commented):
Thanks @rajukumar-cflt for fixing this. Could you please add more details about the problem and the fix in the PR description? Maybe also an example of the execution and the output.

rajukumar-cflt commented:
> Thanks @rajukumar-cflt for fixing this. Could you please add more details about the problem and the fix in the PR description? Maybe also an example of the execution and the output.

- Updated PR description with brief summary of changes
- Added details of script output without and with fix in https://confluentinc.atlassian.net/browse/TRAFFIC-11003
